src/pelib/RichHeader.cpp

```diff
@@ -13,6 +13,686 @@
 
 namespace PeLib
 {
+	// Map of richHeaderProductId -> ProductName
+	const std::vector<std::string> productNames =
```

Comment:
Since we've started using C++17, you can use `std::string_view` instead of `std::string` in case you don't necessarily need `std::string`.

Reply:
Wontfix. If you want to upgrade code to C++ 17, I'll make a separate ticket. All constant strings shall be upgraded to string_view then. It will also require to update CMakeLists.txt for Visual Studio (flag /std:c++17)